### PR TITLE
fix falcon-40b accuracy issue

### PR DESCRIPTION
**Pull request by Yejing-Lai**
"FalconDecoderLayer" module will choose the "glmtype" fused_qkv_type. But the Falcon model should use the "bloomtype".

### CODE REVIEW

**Comment by delock** [comment]
@Yejing-Lai  can you add text output before/after your fix for documentation purpose?  Thanks!

**Comment by Yejing-Lai** [comment]
before fix output: ('DeepSpeed is a machine learning framework for deep neural networks (DNNs) and deep reinforcement learning (DRL). It is written in C++ and is   e e e e e e e e e e e e e e e e e e e e e e e e e e e e e e', 32)

after fix output:('DeepSpeed is a machine learning framework for deep neural networks (DNNs) and deep reinforcement learning (DRL). It is written in C++ and is based on PyTorch.\nDeepSpeed is designed to scale DNNs and DRL to large numbers of GPUs and TPUs.\nDeepSpeed is', 32)

**Comment by delock** [comment]
> before fix output: ('DeepSpeed is a machine learning framework for deep neural networks (DNNs) and deep reinforcement learning (DRL). It is written in C++ and is e e e e e e e e e e e e e e e e e e e e e e e e e e e e e e', 32)
> 
> after fix output:('DeepSpeed is a machine learning framework for deep neural networks (DNNs) and deep reinforcement learning (DRL). It is written in C++ and is based on PyTorch.\nDeepSpeed is designed to scale DNNs and DRL to large numbers of GPUs and TPUs.\nDeepSpeed is', 32)

Thanks!

**Comment by delock** [comment]
@Yejing-Lai is this PR already submitted to upstream?
